src/parse/asp/objects.go

```diff
@@ -915,25 +915,25 @@ func newConfig(state *core.BuildState) *pyConfig {
 	c["TARGET_ARCH"] = pyString(state.TargetArch.Arch)
 	c["BUILD_CONFIG"] = pyString(state.Config.Build.Config)
 
-	loadPluginConfig(state, c)
+	loadPluginConfig(state.Config, state, c)
```

Comment:
This means that we load plugin config when working in the plugin repo. In this case, we make sure that `FoocTool` is loaded. 

```
[PluginDefinition]
name = foo

[Plugin "foo"]
FoocTool = fooc
```